Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/preferences/FlowPreference.kt

```diff
@@ -109,6 +111,7 @@ inline fun <reified T : Any?> SharedPreferences.createFlowPreference(
     writer = FlowPreference.basicWriter()
 )
 
+@Deprecated("Switch to DataStore")
```

Comment:
I think that is not necessary , it makes sonar complains and we are anyway removing it soon, Please remove it